flixel/addons/effects/FlxEffectFade.hx

```diff
@@ -0,0 +1,154 @@
+package flixel.addons.effects;
+
+import flixel.util.FlxColor;
+import flixel.util.FlxDestroyUtil;
+import openfl.display.BitmapData;
+import openfl.geom.Point;
+
+/**
+ * This gradually fills the bitmapData with a color.
+ * 
+ * @author adrianulima
+ */
+class FlxEffectFade implements IFlxEffect
```

Comment:
@Gama11 Do you think it worth send `FlxFadeEffect` together in this pull? Since tween aparently works easier.